<div class="divider"></div>
<div class="section">
  <a class="anchor" name="consortium"></a>
  <!-- <h3>Consortium</h3> -->

  <div class="parallax-container valign-wrapper">
    <div class="section ">
      <div class="container">
        <div class="row center">
          <h1 class="left">Consortium</h1>
        </div>
      </div>
    </div>
    <div class="parallax"><img src="/images/banner2.png" alt="Unsplashed background img 3"></div>
  </div>


  <div class="row">

    <div class="col s12 m6 l6">

      <h4>Scientific partners</h4>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal1" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">

            <img class="responsive-img" src="/images/DTU_logo_text.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal1" class="modal">
        <div class="modal-content">
          <h4>Technical University of Denmark</h4>
          <p>The sequencing, informatics, and modeling (SIM) group led by Drs. Markus Herrgard and Nikolaus Sonnenschein at the Novo Nordisk Foundation Center for Biosustainability at DTU focuses on developing and applying tools for microbial cell factory
            design. SIM group members have developed widely- used modeling platforms such as the COBRA (<a href="https://opencobra.github.io/" target="_blank">opencobra.github.io/</a>) and MASS (<a href="https://opencobra.github.io/MASS-Toolbox/" target="_blank">opencobra.github.io/MASS-Toolbox/</a>) toolboxes. Additionally, Dr. Herrgard has extensive experience in developing web-based
            software for genomic and metagenomic data mining and synthetic biology design from a leading synthetic biology company (Synthetic Genomics, Inc.).</p>
          <h5>Personnel involved:</h5>
          <table>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/markus_herrgard.jpg">
              </td>
              <td>
                <p align="justify"><b>Markus Herrg√•rd</b>, Professor in Data-driven cell factory engineering at the Novo Nordisk Foundation Center for Biosustainability (CFB) at the Technical University of Denmark. He is also the Director of the iLoop Translational Core Unit at the CFB focusing on development of commercialization ready microbial cell factories. Markus has a Ph.D. in Bioengineering from the University of California, San Diego and M.Sc./B.Sc. degrees in Engineering Physics and Mathematics from Aalto University in Finland. From 2006 to 2008, he was a project leader at the University of California, San Diego focusing on systems biology of the yeast S. cerevisiae. From 2008 to 2012 Markus was a senior scientist and group leader at Synthetic Genomics, Inc. in La Jolla, CA leading a group focused on genome mining, synthetic biology design and modeling. He has been at the CFB since 2012 and has been the Director of the iLoop Unit since 2014. Markus is co-author of over 50 peer-reviewed publications with more than 5000 citations and is a co-inventor of several patents and patent applications.</p>
              </td>
            </tr>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/nikolaus_sonnenschein.jpeg">
              </td>
              <td>
                <p align="justify"><b>Nikolaus Sonnenschein</b> is an <em>in silico</em> strain engineer and group leader at <a href="http://www.biosustain.dtu.dk/" target="_blank">The Novo Nordisk Foundation Center for Biosustainability</a> at the <a href="http://dtu.dk/" target="_blank">Technical University of Denmark</a>. He is involved in the <a href="http://www.biosustain.dtu.dk/english/Research/Cores" target="_blank">iLoop</a><a href="#fn2"><sup>2</sup></a> project that brings together people from genome engineering, informatics, modeling, robotics, fermentation, screening, and analytics to develop an accelerated strain engineering process.
Before moving to Denmark, he worked as a postdoc in the <a href="http://systemsbiology.ucsd.edu/" target="_blank">Systems Biology Research Group</a> at <a href="http://ucsd.edu/" target="_blank">University of California, San Diego</a>.
Before going to the US, he received a PhD in Bioinformatics from <a href="http://jacobs-university.de/" target="_blank">Jacobs University Bremen</a> in Germany.</p>
              </td>
            </tr>        
            <tr valign="top">
              <td>
                <img class="picture" src="/images/ricarda_lohmann.png">
              </td>
              <td>
                <p align="justify"><b>Ricarda Lohmann</b>, has a degree in Political Science, Communication and Law (Technical University Dresden, Lund University and FSU Jena). She was a project manager at the European Project Center of TU Dresden and is now working as project assistant at the Center for Biosustainability (Technical University of Denmark). Her work focusses on the management of EU-projects (H2020, Tempus, LLP) and impact assessment tools.</p>
              </td>
            </tr>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/Elisabeth_knudsen.jpg">
              </td>
              <td>
                <p align="justify"><b>Elisabeth Beck Knudsen</b>, has a degree in Social Sciences and Global Development (Roskilde University and Copenhagen University). She has worked as a project manager on several EU-projects at The National Space Institute (DTU Space) and is now working as a project assistant at the Center for Biosustainability (Technical University of Denmark). Elisabeth is the maternity cover for Ricarda Lohman.</p>
              </td>
            </tr>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/svetlana_galkina.jpg">
              </td>
              <td>
                <p align="justify"><b>Svetlana Kutuzova</b> joined the project at May 2016 in a role of software developer with main focus on designing and developing fast, reliable and easy-to-use web platform which will provided access to advanced bioinformatics algorithms. She has Masters degree in Applied Mathematics and Computer Science from Lomonosov Moscow State University and has previously worked as software developer in different domains, including e-commerce and weather forecasting.</p>
              </td>
            </tr>                   
            <tr valign="top">
              <td>
                <img class="picture" src="/images/Moritz_Beber.jpg">
              </td>
              <td>
                <p align="justify"><b>Moritz Beber</b> joined the DD-DeCaF team as a post-doc in January 2017. He holds a PhD in bioinformatics and has previously researched the topology of metabolic networks, large scale dynamics of transcriptional regulation in E. coli, comparisons of manufacturing  and metabolic systems, and extracting features from meta studies of toxicogenomics data. Moritz is passionate about delivering useful, well designed, and enjoyable tools to experimentalists and other developers alike. He will ensure that the DD-DeCaF platform adequately represents biological phenomena and that the individual services can be employed for bioinformatics use.</p>
              </td>
            </tr>
             <tr valign="top">
              <td>
                <img class="picture" src="/images/christian_lieven.jpg">
              </td>
              <td>
                <p align="justify"><b>Christian Lieven</b> joined DD-DeCaF as a post-doctoral researcher in March 2018. He received his PhD working on the Environmentally Friendly Proein Production (EFPro2) project, for which he reconstructed a genome-scale metabolic model for the methanotroph Methylococcus capsulatus. Furthermore, by engaging the COBRA community, Christian and colleagues developed `memote`, a software for quality control of genome-scale metabolic models inspired by common software development practises. Christian is interested in emerging technologies to harness the potential of C1 feedstocks, software development, and the standardization and maintenance of genome-scale metabolic reconstructions. He seeks to create meaningful tools that enhance rather than impede the work of the biotech community through an enjoyable and self-explanatory interface.</p>
              </td>
            </tr>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/Ali_Kaafarani.jpeg">
              </td>
              <td>
                <p align="justify"><b>Ali Kaafarani</b> joined the project in December 2017 as a software developer and operations engineer. He focuses on correctness, robustness and resilience in the DD-DeCaF platform, delivering a stable, secure and trustworthy experience. Ali has a B.E. in Computer Science from The Arctic University of Norway, where his final year project involved processing and presentation of satellite data from NASAs Direct Readout Laboratory. Before coming to Denmark, he worked as a systems developer at The Norwegian Trekking Association, promoting mountain hiking and outdoors activities.</p>
              </td>
            </tr>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/henning_redestig.jpg">
              </td>
              <td>
                <p align="justify"><i><b>Henning Redestig</b>. Henning has contributed to the project until December 2017. We wish him good luck for his future endeavors.</i></p>
              </td>
            </tr>
            <tr valign="top">
              <td>
                <img class="picture" src="/images/danny_dannaher.jpeg">
              </td>
              <td>
                <p align="justify"><i><b>Danny Dannaher</b>. Danny has contributed to the project until September 2017. We wish him good luck for his future endeavors.</i></p>
              </td>
            </tr>
          </table>

          <div class='center-align'>
            <a class="btn" href="http://www.biosustain.dtu.dk/english" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal2" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/EMBL_logo.png" alt="" style="width: 70%">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal2" class="modal">
        <div class="modal-content">
          <h4>European Molecular Biology Laboratory</h4>
          <p>PIs, P. Bork and K. Patil, have complementary research expertise and have together considerable experience in bioinformatics, specifically in function prediction, metagenomics and modeling. They have successfully delivered in >10 European consortia
            in FP6 and FP7 including consortia focusing on analysis of metagenomics data relevant for human health and industrial applications. They have also developed several widely-used bioinformatics tools such as <a href="https://string.embl.de/" target="_blank">String</a> and 
            <a href="https://pathways.embl.de/" target="_blank">iPath</a>.
          </p>
          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/kiran_patil.jpeg">
              </td>
              <td>
                <p align="justify"><b>Kiran Patil</b>. M. tech. (Chemical engineering) 2002, Indian Institute of Technology, Bombay. PhD (Systems biology) 2006, then Assistant Professor, 2006‚Äì2010, Technical University of Denmark. Group leader at EMBL since 2010.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/peer_bork.jpg">
              </td>
              <td>
                <p align="justify"><b>Peer Bork</b> is senior group leader and joint head of the Structural and Computational Biology unit at EMBL, a European research organization with headquarters in Heidelberg where he also serves as strategic head of bioinformatics. In addition, he holds an appointment at the Max-Delbr√ºck-Center for Molecular Medicine in Berlin.<br>

Dr. Bork received his PhD in Biochemistry (1990) and his Habilitation in Theoretical Biophysics (1995). He works in various areas of computational and systems biology with a focus on function prediction, comparative analysis and data integration. He has published more than 500 research articles in international, peer-reviewed journals, among them more than 60 in Nature, Science or Cell. <br>

According to ISI (analyzing 10 years spans), Dr. Bork was for many years the most cited European researcher in Molecular Biology and Genetics and is among the top 5 worldwide in Biochemistry and Biology. He is on the editorial board of a number of journals including Science and functions as senior editor of the journal Molecular Systems Biology.<br> 

Dr. Bork co-founded five successful biotech companies, two of which went public. More than 35 of his former associates now hold professorships or other group leader positions in prominent institutions all over the world. He received the "Nature award for creative mentoring" for his achievements in nurturing and stimulating young scientists and was recipient of the prestigious "Royal Society and Acad√©mie des Sciences Microsoft award" for the advancement of science using computational methods.  Dr. Bork obtained two competitive ERC advanced investigator grants and is elected member of both the German national academy of sciences (Leopoldina) and the European molecular biology organization (EMBO).</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/jaime_h_cepas.jpeg">
              </td>
              <td>
                <p align="justify"><b>Jaime Huerta-Cepas</b> has a degree in Biological Sciences from the Universidad Complutense of Madrid and a PhD on Molecular Biology and Evolution (CNIO and CIPF; Madrid, Valencia, 2004-2008). He worked as a postdoc at CRG (Barcelona, 2009-2014), and currently holds a staff research scientist position at EMBL (Heidelberg). His past and current research lines include i) studying the phylogenetic variability among gene families and its impact on the species taxonomy and the reconstruction of the Tree of Life ii) studying the role of gene duplication on the acquisition of novel gene functions through the use of phylogenetic and phylostratigraphic techniques iii) predicting orthology relationships between molecular sequences iv) studying horizontal gene transfer events in prokaryotes and other microbial organisims v) characterizing the genetic variability within and between microbial communities in environmental and gut metagenomic samples vi) discovering and characterizing novel gene functions and unknown organisms out of massive metagenomic data, with a focus on the discovery of novel enzymes (responsible for DD-DeCaF WP2).</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/daniel_machado.jpeg">
              </td>
              <td>
                <p align="justify"><b>Daniel Machado</b> has degree in Mathematics and Computer Science (University of Minho, 2006) and a PhD in Bioengineering (MIT-Portugal program, 2012). He was a post-doc at the Novo Nordisk Foundation Center for Biosustainability (Denmark), and is currently a post-doc at EMBL (Germany). His work focuses on the development of computational approaches for analysis and simulation of metabolism.  His current interests include the study of microbial communities and the development of software tools for generation and simulation of microbial community models from large-scale multi-omics datasets.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/sergej_andrejev.jpeg">
              </td>
              <td>
                <p align="justify"><b>Sergej Andrejev</b> is a scientific programmer in EMBL, Heidelberg. In his research he applies metabolic modeling, statistical analysis and other bioinformatics technics to study microbial population ecology dynamics. Recently he co-authored a paper elucidating the role of bacterial metabolic cooperation in bacterial ecology. The study demonstrated that competition is not the only major force shaping bacterial communities and metabolic cooperation is driving force behind many naturally occurring communities. His current area of interest includes metabolic interactions and population dynamics in soil, gut, ocean and others bacterial communities as well as single cells interactions in multicellular organisms.</p>
              </td>
            </tr>           
            <tr>
              <td>
                <img class="picture" src="/images/luis_coelho.jpeg">
              </td>
              <td>
                <p align="justify"><b>Luis Pedro Coelho</b> has BS and MS degrees in Computer Science (IST, University of Lisbon, 2004 and 2006) and a PhD in computational biology from Carnegie Mellon University (2011). He is currently a postdoc at EMBL (Germany) work in the group of Peer Bork. His current work is on large-scale microbial ecology, with a focus on the use of sequencing technologies, in particular developing computational tools capable of integrating very large metagenomics datasets</p>
              </td>
            </tr>
             <tr>
              <td>
                <img class="picture" src="/images/paula_jouhten.jpeg">
              </td>
              <td>
                <p align="justify"><i><b>Paula Jouhten</b>, Paula has contributed to the project until December 2017. We wish her good luck for her future endeavors.</i></p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="http://www.embl.de/" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal3" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/Chalmers.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal3" class="modal">
        <div class="modal-content">
          <h4>Chalmers University of Technology</h4>
          <p>The J. Nielsen‚Äôs group at Chalmers is one of the world leaders on genome-scale modeling of microorganisms, and statistical and model-based analysis of omics data for cell factory and human health applications. The group has reconstructed >20
            models and has developed software tools for performing semi-automatic reconstruction. The group has developed several tools for simulation and integrative analysis of omics data in the context of genome-scale models (GEMs) as well as general
            omics analysis tools (<href="biomet-toolbox.org" target="_blank">).</p>
          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/hongzhong.jpeg">
              </td>
              <td>
                <p align="justify"><b>Hongzhong Lu</b>,¬†PhD Bioengineering in 2016 (East China University of Science and Technology), has one year of work experience in DSM biotechnology center (Shanghai) after graduation. In his PhD phase, he mainly focused on reconstruction of Aspergillus niger genome metabolic model, 13C labelled flux analysis and multi-omics analysis based on model simulation. He is interested in the big data mining based on genome scale metabolic model.¬† He also has great interest in the reconstruction of advanced metabolic models and their innovative applications, like the ME, GEMs-PRO, etc. He is currently involved in the DD-DeCaF project to develop yeast genome metabolic model of high quality.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/Yu.jpg">
              </td>
              <td>
                <p align="justify"><b>Yu Chen</b>,¬†PhD student and will receive his PhD in Bioengineering in 2017 (East China University of Science and Technology). His research interest is in systems biology of metabolism. In his previous research, he applied constraint-based methods, especially flux balance analysis, to study physiology and metabolic regulation of industrial microorganisms. Besides, he focused on integrative analysis of omics data using genome-scale metabolic models. He is currently involved in the DD-DeCaF project to develop integrated models of metabolism and protein synthesis.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/BenjaminBarja.jpg">
              </td>
              <td>
                <p align="justify"><b>Benjam√≠n S√°nchez</b>,¬†PhD student and computational biologist with an MSc degree in Biotechnology Engineering from the Pontifical Catholic University of Chile. He has interests in systems biology and mathematical modeling of biological processes, and as of 2014 he is a PhD student in the Division of Systems and Synthetic Biology in Chalmers. For his thesis, he is working on genome scale modeling of yeast, developing new tools for integration of models with omic data and for computational prediction of metabolic engineering strategies. In the DD-DeCaF project he is developing metabolic models that account for protein abundances as constraints.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/feiran_li.jpeg">
              </td>
              <td>
                <p align="justify"><b>Feiran Li</b>,¬†PhD student with a MSc degree in Biochemical Engineering (Tianjin University, China). From 2017, she has been a PhD student in the Division of Systems and Synthetic Biology in Chalmers. She is working on genome scale modelling of yeast, with a focus on modelling of industrial proteins production and secretion through a combination of metabolic models and protein secretory models. She is currently involved in the DD-DeCaF project to develop integrated models of metabolism and protein secretion.</p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="http://www.sysbio.se/Lab_Nielsen/lab_nielsen.html" target="_blank">Partner website</a>
          </div>
          <div class="modal-footer">
            <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
          </div>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal4" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/EPFL.png" alt="" style="width: 70%">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal4" class="modal">
        <div class="modal-content">
          <h4>√âcole polytechnique f√©d√©rale de Lausanne</h4>
          <p>The Hatzimanikatis group has 20 years of experience in academic and industrial research for the design of metabolic pathways and whole-cell industrial biocatalysts. Dr. Ljubisa Miskovic is a senior research scientist with extensive experience
            with modeling and analysis of metabolic pathways. The group has developed large-scale nonlinear models for S. cerevisiae, human cells, E. coli and P. putida, which account for the thermodynamic and kinetic properties of metabolic pathways
            and can be used for the identification of rate-limiting steps in the absence of complete kinetic information on the pathway enzymes.
          </p>
          <h5>Personnel involved:</h5>
          <table>
             <tr>
              <td>
                <img class="picture" src="/images/VassilyHatzimanikatis.jpg">
              </td>
              <td>
                <p align="justify"><b>Vassily Hatzimanikatis</b>,¬†has been Associate Professor of Chemical Engineering and Bioengineering at the √âcole Polytechnique F√©d√©rale de Lausanne (EPFL) since 2006. He received his Diploma in Chemical Engineering from the University of Patras (Greece) in 1991. In 1997, he graduated from the California Institute of Technology (Caltech) with a master (M.S.) and a doctoral (Ph.D.) degree in Chemical Engineering under the supervision of Prof. Jay Bailey. His work focused on the mathematical analysis of metabolic reaction networks and their design to achieve desired phenotypes.
Professor Hatzimanikatis continued his career as a research associate in the laboratory of Prof. Jay Bailey in the Swiss Federal Institute of Technology (ETH) in Zurich (Switzerland). Between 1997 and 2000, he worked on the development of biocatalysts for the production of industrial chemicals in DuPont, Cargill, and Cargill Dow. In 2000, he became Assistant Professor of Chemical Engineering at Northwestern University in Evanston, Illinois. He held this position until he joined the EPFL.
Professor Hatzimanikatis‚Äô research interests are in the areas of systems biotechnology, bioinformatics, and metabolic engineering. He is editor in chief of¬†Metabolic Engineering Communications¬†and associate editor of¬†Biotechnology and Bioengineering, and¬†Metabolic Engineering. He has published over 100 technical articles and he is co-inventor in three patents.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/GeorgiosFengos.jpg">
              </td>
              <td>
                <p align="justify"><b>Georgios Fengos</b>¬†is a Post-Doc researcher at EPFL and joined the DD-Decaf project in 2017. He holds a PhD in Systems Biology from ETH Zurich, and a Diploma in Chemical Engineering from University of Patras in Greece. His research interests are in the area of Systems Biotechnology with focus on the generation and analysis of large-scale kinetic models of metabolic networks for the design of metabolic engineering strategies.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/liliana_angeles.jpg">
              </td>
              <td>
                <p align="justify"><b>Liliana Angeles</b>¬†is a Post-Doc researcher at EPFL. She holds a PhD in Chemical Engineering from the University of Manchester (UK), and an MSc in Bioprocesses from the National Polytechnic Institute (Mexico). Her research interests lie in the field of systems biology and bioengineering, with focus on the modelling of metabolic networks and complex reaction-diffusion systems.</p>
              </td>
            </tr>                   
            <tr>
              <td>
                <img class="picture" src="/images/ZhalehHosseini.jpg">
              </td>
              <td>
                <p align="justify"><b>Zhaleh Hosseini</b>,¬†PhD student with an MSc degree in Biotechnology from University of Tehran. In her MSc thesis, she worked on gap filling of metabolic network models using gene expression data. As of June 2016, she is a PhD student in the Laboratory of Computational Systems Biotechnology in EPFL. She is working on development of optimization methods for analysis of thermodynamically constrained metabolic network models.</p>
              </td>
            </tr>            
            <tr>
              <td>
                <img class="picture" src="/images/daniel_weilandt.jpeg">
              </td>
              <td>
                <p align="justify"><b>Daniel Robert Weilandt</b>, PhD student with an MSc degree in Microsystem Engineering from the Albert-Ludwigs University of Freiburg. In his PhD thesis he was working on dissipative particle models for cell membranes. He started in May 2016 as a PhD student in the Laboratory of Computational Systems Biotechnology in EPFL. He now works on methods to integrate enzyme data into large scale metabolic models.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/robin_eriksson.jpeg">
              </td>
              <td>
                <p align="justify"><b>Robin Denhardt-Eriksson</b>, PhD at student at the Laboratory of Computational Systems Biotechnology (LCSB) since 2017. He holds an MsC in Chemistry and Chemical Engineering from EPFL. His research interests are in the area of analysis and understanding of large-scale kinetic models. </p>
              </td>
            </tr>
             <tr>
              <td>
                <img class="picture" src="/images/MericAtaman.jpg">
              </td>
              <td>
                <p align="justify"><i><b>Meric Ataman</b>. Meric has contributed to the project until November 2017. We wish him good luck for his future endeavors.</i></p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="http://lcsb.epfl.ch/" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal5" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/CEB_logotipo.png" alt="" style="width: 65%">
            <img class="responsive-img" src="/images/um.jpg" alt="" style="width: 30%">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal5" class="modal">
        <div class="modal-content">
          <h4>University of Minho</h4>
          <p>The Biosystems ‚Äì Systems Biology and Metabolic Engineering research group, led by Prof. Isabel Rocha has currently over 20 members working in the field of Systems Biology and Metabolic Engineering, with emphasis on the development of algorithms
            for strain design. Moreover, the team has expertise and experience in developing open-source user-friendly software tools such as Merlin (<a href="http://www.merlin-sysbio.org" target="_blank">www.merlin-sysbio.org</a>) and OptFlux (<a href="http://www.optflux.org" target="_blank">www.optflux.org</a>).
          </p>
          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/IsabelRocha.jpg">
              </td>
              <td>
                <p align="justify"><b>Isabel Rocha</b>,¬†teaches Bioinformatics and Systems Biology at Minho University where she is the PI of several projects in Systems Biology applied to Industrial Biotechnology and Life Sciences, with more than 100 papers published in international journals, books and international conferences. She did her PhD in Chemical and Biological Engineering at Minho University in Portugal and a Post-Doc in Systems Biology at the Technical University of Denmark. She spent one semester at MIT in Boston and she is part of the MIT-Portugal Program Faculty, being the head of a National program in Teaching innovation and entrepreneurship. She is co-founder and Scientific Director of both SilicoLife and Biotempo. Biotempo is focused on Bioprocess Development for the food industry while SilicoLife develops improved microbial strains for industrial applications using in silico methods.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/Miguel_Rocha.jpg">
              </td>
              <td>
                <p align="justify"><b>Miguel Rocha</b>,¬†is an Associate Professor at the Department of Informatics of the University of Minho, being the Director of the Master course in Bioinformatics, and teaching contents related to Bioinformatics and Data Sciences/ Machine Learning. He is also a senior researcher at the Centre of Biological Engineering, performing research in the themes Bioinformatics and Systems Biology. He has (co-)authored over 150 peer-reviewed articles, and participated in over 20 funded research projects (5 as a PI). He also coordinates several open-source software development projects including OptFlux and @Note.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/odias.jpeg">
              </td>
              <td>
                <p align="justify"><b>√ìscar Dias</b>,¬†is an Invited Assistant Researcher and Professor at the Centre of Biological Engineering at the University of Minho. He completed a Doctoral Program in Chemical and Biological Engineering by the University of Minho in 2013. Before, he completed a Master's degree in Informatics and a Licentiate's degree in Biological Engineering. He (co-)authored over 20 peer-reviewed publications and participated in several funded research projects. He is the main developer of the open-source model's reconstruction platforms <i>merlin</i> and <i>triage</i>.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/SaraCorreia.jpg">
              </td>
              <td>
                <p align="justify"><b>Sara Correia</b>,¬†post-doc researcher at the Centre of Biological Engineering (CEB- University of Minho). PhD in Informatics (University of Minho, 2016), M.Sc in Bioinformatics (University of Minho, 2011) and BSc in Mathematics and Computer Science (University of Minho, 2001).
She worked as software developer  between 2001 and 2005 and as a high school teacher in the following five years. During the PhD, she worked in the development of computational tools for the reconstruction of tissue-specific metabolic models. 
She joined the DD-DeCaF project, in September 2016, to develop optimization algorithms for the design of optimal strains and microbial communities.</p>
              </td>
            </tr>            
            <tr>
              <td>
                <img class="picture" src="/images/sophia_santos.jpeg">
              </td>
              <td>
                <p align="justify"><b>Sophia Santos</b>,¬†is currently a PhD candidate in the MIT-Portugal doctoral program in Bioengineering at Centre of Biological Engineering (CEB - University of Minho). M.Sc in Bioinformatics (University of Minho, 2013), B.Sc in Biochemistry (University of Porto, 2011) and B.Sc in Mathematics (University of Minho, 2007). She has worked on the development of computational and experimental methods for measuring biomass composition and evaluating its impact in genome-scale models predictions and also in the integration of yeast genome-scale metabolic models. She joined the DD-DeCaF project, in July 2016, to develop optimization tools for the design of microbial communities and the to apply those tools to some case studies.</p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="http://www.ceb.uminho.pt/biosystems" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

    </div>

    <div class="col s12 m6 l6">

      <h4>Industrial partners</h4>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal6" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/silicolife.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal6" class="modal">
        <div class="modal-content">
          <h4>SilicoLife</h4>
          <p>SilicoLife creates computational biology solutions for the fast growing Industrial Biotechnology applications, allowing our clients to design optimized microbial strains for the cost-effective production of specific target compounds such as biofuels, chemicals, food ingredients or biopolymers.
          Leading chemical, materials and synthetic biology companies have chosen Silicolife as development partner to maintain their market edge in increasingly competitive markets. SilicoLife is considered a highly innovative company, including as one of the 40 Hottest Small Companies in the Advanced Bioeconomy by BiofuelsDigest 2014/15 and Startup of the week by Wired UK.
          </p>

          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/simao_soares.jpg">
              </td>
              <td>
                <p align="justify"><b>Sim√£o Soares</b>, CEO SilicoLife. MSc Bioinformatics (University of Minho), postgraduate in management from NOVA School of Business and Economics and trained in Blue Ocean Strategy at INSEAD. Was visiting researcher at the Technical University of Denmark, in the integration of omics data with genome scale models. Collaborated with the UMinho in the fields of Bioinformatics and Systems Biology, with published work in computational biology top ranked journals.  Member of the board of P-BIO, Portuguese Bioindustry Association and consultant for international organizations in industrial biotechnology topics.</p>
              </td>
            </tr>
            <tr>
            <tr>
              <td>
                <img class="picture" src="/images/paulo_vilaca.jpg">
              </td>
              <td>
                <p align="justify"><b>Paulo Vila√ßa</b>, COO SilicoLife. MSc in Bioinformatics and is currently developing a PhD in Computer Science. He has a strong background in software development and engineering. Paulo is responsible for the daily management of a development team with more than 15 people working in systems biology and bioinformatics applied to industrial biotechnology problems.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/paulo_maia.jpg">
              </td>
              <td>
                <p align="justify"><b>Paulo Maia</b>, Head of Development. PhD in Bioengineering (MIT Portugal, U.Minho), MSc in Bioinformatics and BSc in Computer Science (U.Minho). His work during the past 9 years ranged from the development of machine learning approaches for feature extraction in biological datasets to the development of novel phenotype prediction and computational strain optimization methods. He is an experienced software engineer, co-developing the OptFlux workbench and several other open-source applications and co-authored various publications in the metabolic engineering arena.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/sonia_carneiro.jpg">
              </td>
              <td>
                <p align="justify"><b>S√≥nia Carneiro</b>, Head of Development. PhD in Chemical and Biological Engineering from the University of Minho, Portugal and a post-doc in Cancer Systems Biology at the Computational Genomics Laboratory in IGC and University of Minho, Portugal. Her work during the past 10 years is focused on the application of Systems Biology approaches for the characterization of metabolic bottlenecks in cellular systems. She is a co-author of several publications in research topics like recombinant bioprocesses, metabolomics and metabolic modelling.</p>
              </td>
            </tr>
          </table>
          
          <div class='center-align'>
            <a class="btn" href="http://www.silicolife.com/" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal7" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/Genialis.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal7" class="modal">
        <div class="modal-content">
          <h4>Genialis d.o.o.</h4>
          <p>Genialis builds data analytics software that facilitates secure and compliant management, rigorous and reproducible pipelines, and intuitive, interactive visualizations. These components are seamlessly integrated into a single system, Genialis Platform, that streamlines the collaboration between end-user and data expert. The existing software focuses on bioinformatics (RNA-seq, microarray, ChIP-seq, whole genome sequencing), however the dataflow engine can accommodate any type of data or algorithm. The Platform leverages cutting-edge web and database technologies, and its modular architecture allows for continuous evolution of sophisticated data analysis methods, including data mining. Genialis d.o.o. was founded in February 2013 in Ljubljana, Slovenia.
          </p>
          <h5>Personnel involved:</h5>
          <table>         
            <tr>
              <td>
                <img class="picture" src="/images/luka_ausec.png">
              </td>
              <td>
                <p align="justify"><b>Luka Ausec</b>, bioinformatician and project manager. PhD in Biotechnology by the University of Ljubljana (2014) with a focus on metagenomic analyses and bioinformatics. Luka joined Genialis in 2015 as a leader of BioBash Workshops programme and designed various courses in bioinformatics and computing for biologists. Currently, he is overseeing the Genialis‚Äô project management team.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/eva_zupa_horacek.png">
              </td>
              <td>
                <p align="justify"><b>Eva Zupan Horaƒçek</b>, designer. Has been a designer for almost 10 years, freelancing for various companies and marketing agencies. She was also worked as junior lecturer at the Faculty of Design (University of Primorska) and as teaching assistant at the Department for landscape architecture at Biotechnical faculty (University of Ljubljana). At Genialis leads the graphic and UX design processes.</p>
              </td>
            </tr>
            <tr>
            <tr>
              <td>
                <img class="picture" src="/images/miha_stajdohar.png">
              </td>
              <td>
                <p align="justify"><b>Miha ≈†tajdohar</b>, CTO, developer and bioinformatician. Phd in Computer and Information Science by the University of Ljubljana (2012) focusing on visualizations and data mining, and Postdoctoral associate Baylor College of Medicine since 2014. Miha has 15 years of experience in software development and data science. After his PhD, Miha co-founded Genialis as Chief Technology Officer, where he leads the software development process.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/nejc_skoberne.png">
              </td>
              <td>
                <p align="justify"><b>Nejc ≈†koberne</b>, CEO. PhD computer and Information Science by the University of Ljubljana (2014). Nejc ≈†koberne has 16 years of experience in the IT industry. He started his career as a system and computer network administrator at Infrax d.o.o. and later continued as an information security expert and auditor at Viris d.o.o.. Nejc finished his PhD in computer science in 2013 and co-founded Genialis, where he is the CEO.</p>
              </td>
            </tr>
             <tr>
              <td>
                <img class="picture" src="/images/matyas_fodor.jpeg">
              </td>
              <td>
                <p align="justify"><b>M√°ty√°s Fodor</b>, obtained his BSc in Molecular Bionics at the P√°zm√°ny P√©ter Catholic University in Hungary. He changed career to become a web developer and gained some industrial experience. He's responsible for maintaining and developing the front end web application of the DD-DeCaf project.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/joao_pita_costa.jpg">
              </td>
              <td>
                <p align="justify"><i><b>Jo√£o Pita Costa</b>. Jo√£o has contributed to the project until March 2017. We wish him good luck for his future endeavors.</i></p>
              </td>
            </tr>
            <tr>
            <tr>
              <td>
                <img class="picture" src="/images/nace_kranjc.png">
              </td>
              <td>
                <p align="justify"><i><b>Nace Kranjc</b>. Nace has contributed to the project until March 2017. We wish him good luck for his future endeavors.</i> </p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="https://www.genialis.com/" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal8" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/Biobyte.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal8" class="modal">
        <div class="modal-content">
          <h4>biobyte solutions GmbH</h4>
          <p>biobyte solutions GmbH is a provider of customized bioinformatics tools. The company also provides various bioinformatics services, from data analysis and integration, to database development and maintenance. biobyte solutions will be involved in the development of data mining pipelines and data management systems.
          </p>
          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/Ivica_L.jpg">
              </td>
              <td>
                <p align="justify"><b>Ivica Letunic</b>,¬†PhD,¬†is a co-founder and managing director of biobyte solutions GmbH. He received his PhD from Heidelberg University and EMBL in 2004. He has extensive experience in the development of data management systems and visualization of biological data. He is the author of several widely used tools, for example iTOL (interactive Tree Of Life), SMART (a protein domain prediction and exploration tool) and iPath (interactive Pathways explorer). Ivica has co-authored over 50 peer-reviewed publications with more than 20 000 citations.</p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="http://www.biobyte.de/" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal9" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/Biosyntia.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal9" class="modal">
        <div class="modal-content">
          <h4>Biosyntia ApS</h4>
          <p>Biosyntia is a synthetic biology and biocatalysis company founded in 2012 by Dr. Hans Genee and Professor Dr. Morten Sommer. The company applies evolutionary metabolic engineering for the development of microbial cell factories of fine chemical products. Biosyntia‚Äôs R&D process involves iterative cycles of pathway design, construction and high throughput testing, and a major challenge is the analysis and interpretation of the ever increasing amounts of data generated. Furthermore, in the search for more efficient pathway routes, Biosyntia functionally interrogates metagenomics for novel genes and pathways. Biosyntia is funded by various EU grants and a seed investment from Novo Seeds.
          </p>

          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/hans_genee.jpg">
              </td>
              <td>
                <p align="justify"><b>Hans Genee</b>,¬†PhD,¬†is co-founder at Biosyntia. He did his doctoral degree in biotechnology at the NNF Center for Biosustainability at the¬†Technical University of Denmark with additional training from Harvard University in the group of Professor George Church and from UC Berkeley / the Joint Bio-Energy Institute in the group of professor Jay Keasling. Hans holds personal awards for Exceptional Achievements (McKinsey & Co. 2011), has authored numerous scientific papers and patents and co-founded Biosyntia.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/luisa_simona_gronenberg.jpg">
              </td>
              <td>
                <p align="justify"><b>Luisa Simona Gronenberg</b>, PhD, is a metabolic engineering scientist at Biosyntia. Luisa holds a B.Sc. with Honors in Chemistry from the University of Arizona and an M.Sc. and PhD in Chemistry and Chemical Biology from Harvard University, where she was a recipient of a National Science Foundation Graduate Research Fellowship and worked in the laboratory of Prof. Daniel Kahne. Prior to joining Biosyntia, Luisa spent three years as a postdoctoral fellow in the Metabolic Engineering and Synthetic Biology Laboratory at UCLA, where she led an ARPA-E-funded research team in engineering novel carbon fixation cycles into bacteria.</p>
              </td>
            </tr>
          </table>

          <div class='center-align'>
            <a class="btn" href="http://biosyntia.com//" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

      <!-- Modal Trigger -->
      <a class="modal-trigger" href="#modal10" style="color: black;max-width:70%">
        <div class="card-panel hoverable">

          <div class="card-content">
            <img class="responsive-img" src="/images/DSM.png" alt="">
          </div>
        </div>
      </a>
      <!-- Modal Structure -->
      <div id="modal10" class="modal">
        <div class="modal-content">
          <h4>DSM</h4>
          <p>DSM is a global science-based company with application areas ranging from food, feed and pharma to bio-based chemicals and bio-based fuels. DSM employs rational approaches towards metabolic and protein engineering using bioinformatics, systems biology and genetic engineering tools. Dr. Hans Roubos and Dr. Liang Wu ‚Äì both senior bioinformaticians at the DSM Biotechnology R&D Center ‚Äì have over 10 years of experience in developing industrial processes using bacteria, yeast and filamentous fungi within DSM and are closely connected to the internal high-throughput wet lab activities.
          </p>
          <h5>Personnel involved:</h5>
          <table>
            <tr>
              <td>
                <img class="picture" src="/images/hans_roubos.jpg">
              </td>
              <td>
                <p align="justify"><b>Hans Roubos</b>,¬†PhD,¬†is Principal Scientist Bioinformatics & Modeling at the DBC. Hans holds a MSc in Bioprocess Engineering (Wageningen University) and PhD in Biotechnology & Systems and Control Engineering (Delft University of Technology). Hans joined DSM in 2003 and currently manages the Bioinformatics & Modeling group at DBC. This team supports multi-disciplinary projects with DNA construct design, protein engineering, metabolic pathway engineering, omics data mining, sequencing, and biostatistics. In addition, Hans manages a project on automated strain engineering technology. Hans published over 20 peer reviewed articles and contributed to 7 book chapters.</p>
              </td>
            </tr>
            <tr>
              <td>
                <img class="picture" src="/images/liang_wu.jpg">
              </td>
              <td>
                <p align="justify"><b>Liang Wu</b>, PhD, is Senior Scientist in the Bioinformatics & Modeling group at DBC. Liang holds a MSc in Chemical Engineering (Delft University of Technology) and PhD in Biotechnology (Delft University of Technology) before joining DSM in 2006. Liang‚Äôs main expertise is metabolic engineering, systems and synthetic biology, and modeling, applied in many projects to enable and accelerate the development of optimized strains, processes, and applications at an industrial scale. Liang (co-)authored 15 peer reviewed articles, 1 book chapter, and is inventor of 16 patent applications.</p>
              </td>
            </tr>
          </table>
          <div class='center-align'>
            <a class="btn" href="http://www.dsm.com/corporate/home.html" target="_blank">Partner website</a>
          </div>
        </div>
        <div class="modal-footer">
          <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
        </div>
      </div>

    </div>
  </div>

</div>
